app/src/main/java/com/codesoom/assignment/application/ProductCommandService.java

```diff
@@ -0,0 +1,78 @@
+package com.codesoom.assignment.application;
+
+import com.codesoom.assignment.domain.Product;
+import com.codesoom.assignment.exception.ProductNotFoundException;
+import com.codesoom.assignment.repository.ProductRepository;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+
+import java.util.List;
+
+/**
+ * 상품 생성/수정/삭제를 처리합니다
+ */
+@Service
+@Transactional
+public class ProductCommandService {
+
+    private final ProductRepository productRepository;
+
+    public ProductCommandService(ProductRepository productRepository) {
+        this.productRepository = productRepository;
+    }
+
+    /**
+     * 상품을 생성하여 반환합니다.
+     * @param product 생성할 상품
+     * @return 생성된 상품
+     */
+    public Product create(Product product) {
```

Comment:
여기에서 이걸 사용할 수도 있을 거에요.
```suggestion
    public Product create(ProductUpdateRequest product) {
```